### PR TITLE
ADEN-9817 Enable tracking on UCP

### PR DESCRIPTION
**Pull request by Bielik20**

### CODE REVIEW

File: @types/ads/index.d.ts [hunk 1]
```diff
@@ -1,9 +1,48 @@
-interface Ads {
+interface MediaWikiAds {
```
Comment (dmnsobczak) [inline @types/ads/index.d.ts]: This object is also available on Futhead on Muthead and there is no Media Wiki there. FYI.

File: platforms/ucp/setup/context/targeting/ucp-targeting.setup.ts [hunk 2]
```diff
@@ -28,7 +28,7 @@ export class UcpTargetingSetup implements TargetingSetup {
 			s0: wiki.targeting.mappedVerticalName,
 			s1: this.getRawDbName(wiki),
 			s2: this.getAdLayout(wiki),
-			skin: 'ucp',
+			skin: 'oasis',
```
Comment (dmnsobczak) [inline platforms/ucp/setup/context/targeting/ucp-targeting.setup.ts]: If parameter is defined in `ad-context.ts` file then why not remove them from here?

File: @types/ads/index.d.ts [hunk 3]
```diff
@@ -1,9 +1,48 @@
...
+	targeting: MediaWikiAdsTargeting;
+}
+
+interface MediaWikiAdsOpts {
+	isAdTestWiki: boolean;
+	pageType: string;
+	showAds: boolean;
+}
+
+interface MediaWikiAdsTargeting {
```
Comment (dmnsobczak) [inline @types/ads/index.d.ts]: 💎

File: @types/ads/index.d.ts (hunk 1 above)
Comment (Bielik20) [reply @types/ads/index.d.ts]: I know... those types are a mess. I will have to refactor it later.

File: platforms/ucp/setup/context/targeting/ucp-targeting.setup.ts (hunk 2 above)
Comment (Bielik20) [reply platforms/ucp/setup/context/targeting/ucp-targeting.setup.ts]: We are aiming to have a single source of that information - aka remove it from ad-context.

**Pull request merged**